pydriller/repository.py

```diff
@@ -204,6 +204,10 @@ def _prep_repo(self, path_repo: str) -> Generator[Git, None, None]:
                 # git directories because of read-only files.
                 # In this case, just ignore the errors.
                 shutil.rmtree(self._tmp_dir.name, ignore_errors=True)
```

Comment:
Now you need to delete from line 202 to line 206 👍

Reply:
Sorry about that. I forget... 😅